.buildkite/scripts/common.sh

```diff
@@ -69,3 +71,32 @@ with_Terraform() {
     chmod +x ${WORKSPACE}/terraform
     terraform version
 }
+
+google_cloud_auth() {
+    secretFileLocation=${WORKSPACE}/$(mktemp -d -p . -t "${TMP_FOLDER_TEMPLATE_BASE}.XXXXXXXXX")/google-cloud-credentials.json
+    echo "${PRIVATE_CI_GCS_CREDENTIALS_SECRET}" > ${secretFileLocation}
+    gcloud auth activate-service-account --key-file ${secretFileLocation} 2> /dev/null
+    export GOOGLE_APPLICATIONS_CREDENTIALS=${secretFileLocation}
+}
+
+upload_packages_to_gcp_backet() {
+    pattern=${1}
+    baseUri="gs://${JOB_GCS_BUCKET}/${REPO}"
+    bucketUriCommit="${baseUri}"/commits/${BUILDKITE_COMMIT}
+    bucketUriDefault="${baseUri}"/snapshots
+
+    if [[ ${BUILDKITE_PULL_REQUEST} != "false" ]]; then
+        bucketUriDefault="${baseUri}"/pull-requests/pr-${BUILDKITE_COMMIT}
+    fi
+
+    for backetUri in "${bucketUriCommit}" "${bucketUriDefault}"; do
+        gsutil -m -q cp -a public-read -r ${pattern} "${backetUri}"
```

Comment:
nit. backet -> bucket 
```suggestion
    for bucketUri in "${bucketUriCommit}" "${bucketUriDefault}"; do
        gsutil -m -q cp -a public-read -r ${pattern} "${bucketUri}"
```

Reply:
:) fixed.